House-of-glass/src/components/Cards/Horizontal/HorizontalCard.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import Ratings from '../../Ratings/Ratings'
+
+export default function HorizontalCard({CardTitle,ItemPrice,ItemDiscount,DeliveryType,ItemImage,ItemRating}) {
```

Comment:
When there are so many items in the props destructure the props later.

```JSX
const abcd = (props) => {
   const {prop1, prop2} = props
}
```

Reply:
Noted. Will be doing that. 
Thank You.